tests/in-vm.yml

```diff
@@ -0,0 +1,80 @@
+# support use cases:
+#  * testing an image built locally in current environment (set pull variable to false)
+#  * testing an image built locally inside a VM (supply vm_image variable)
+#  * testing an image present in registry in current environment (set pull to true)
+#  * testing an image present in registry inside a VM (set pull and vm_image variables)
+---
+- name: Integration tests for tools container image executed in current environment
+  hosts: localhost
+  vars:
+    # don't pull the test subject by default
+    pull: false
+
+    # don't set up the environment by default (instal and start container runtime)
+    setup: false
+
+    required_packages:
+     - python3-conu
+     - python3-pytest
+
+    tests:
+    - integration
```

Comment:
Can you please describe it? It is called like integration? More comments are welcome for understanding dummies.

Reply:
https://fedoraproject.org/wiki/CI/Tests
https://github.com/TomasTomecek/tools/tree/b497b5ff0d8138d019d4f705c102597ee522128a/tests/integration

Reply:
But you're right that comment is missing, will add.